tests/FSharp.Compiler.ComponentTests/Conformance/BasicGrammarElements/MemberDefinitions/MethodsAndProperties/MethodsAndProperties.fs

```diff
@@ -81,7 +81,23 @@ module MemberDefinitions_MethodsAndProperties =
         |> verifyCompile
         |> shouldFail
         |> withDiagnostics [
-            (Error 827, Line 6, Col 12, Line 6, Col 27, "This is not a valid name for an active pattern")
+            (Error 827, Line 6, Col 15, Line 6, Col 24, "'(|Foo|Bar|)' is not a valid method name. Use a 'let' binding instead.")
```

Comment:
Do I get it right that these are "side-effects" (positive ones) of the changes you've made in CzechExpressions?

Reply:
Yes